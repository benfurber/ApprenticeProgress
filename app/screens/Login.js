// @flow

import React, { Component } from "react";
import { Content, Text } from "native-base";

import { Background } from "components";

type Props = {};

class Login extends Component<Props> {
  render() {
    return (
      <Background>
        <Content>
          <Text>Blank Login</Text>
        </Content>
      </Background>
    );
  }
}

export { Login };
