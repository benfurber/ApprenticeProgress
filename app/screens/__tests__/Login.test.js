// @flow

import React from "react";
import { shallow } from "enzyme";

import { Login } from "../Login";

describe("Login", () => {
  it("has a valid snapshot", () => {
    const screen = shallow(<Login />);

    expect(screen).toMatchSnapshot();
  });
});
