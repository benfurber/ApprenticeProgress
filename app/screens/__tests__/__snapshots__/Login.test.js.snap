// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login has a valid snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Login />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <Styled(Content)>
        <Styled(Text)>
          Blank Login
        </Styled(Text)>
      </Styled(Content)>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Styled(Text)>
          Blank Login
        </Styled(Text)>,
        "virtual": undefined,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "Blank Login",
          "virtual": undefined,
        },
        "ref": null,
        "rendered": "Blank Login",
        "type": [Function],
      },
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Styled(Content)>
          <Styled(Text)>
            Blank Login
          </Styled(Text)>
        </Styled(Content)>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": <Styled(Text)>
            Blank Login
          </Styled(Text)>,
          "virtual": undefined,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Blank Login",
            "virtual": undefined,
          },
          "ref": null,
          "rendered": "Blank Login",
          "type": [Function],
        },
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
